src/pages/LandingPage/LandingPage.jsx

```diff
@@ -1,15 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react'; // <-- useState is a react hook and imported here, you must always import react hooks
 import scrollToElement from '../../shared/utility/scrollToElement';
 import PortfolioCard from './PortfolioCard';
 import ContactForm from './ContactForm';
 import Render3DShape from './Render3DShape';
 
+const POSSIBLE_SHAPES = [
+  'torus',
+  'sphere',
+  'dodecahedron',
+  'icosahedron',
+  'octahedron',
+  'tetrahedron',
+  'torusknot',
+  'sahilsknot'
+];
+
+const INITIAL_SHAPE_INDEX = 0;
+const INITIAL_SHAPE = POSSIBLE_SHAPES[INITIAL_SHAPE_INDEX];
+
 const LandingPage = () => {
   const heroImage = 'hero.jpg';
+  const [shape, setShape] = useState(INITIAL_SHAPE);
+  let prevShapeIndex = INITIAL_SHAPE_INDEX;
 
+  const handleClick = () => {
+    let randomIndex = prevShapeIndex;
+    while (randomIndex === prevShapeIndex) {
+      randomIndex = Math.floor(Math.random() * POSSIBLE_SHAPES.length);
+    }
+    prevShapeIndex = randomIndex;
+    setShape(POSSIBLE_SHAPES[randomIndex]);
+  };
```

Comment:
Here i just cleaned up by removing the console.log and old comment (this is something we always do before merging to main. Make it work -> make it nice (refactor and clean up) -> test & deploy)

I also changed the name chooseRandomShape to randomIndex to better reflect what it actually does (gives you a random index). But again, this was my mistake to begin with.